front/app/containers/Admin/projects/project/analysis/InputPreview/components/CustomFieldLongFieldValues/CheckboxLongField.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Box, Title, Text, Checkbox } from '@citizenlab/cl2-component-library';
+
+import { IIdeaCustomField } from 'api/idea_custom_fields/types';
+
+import T from 'components/T';
+
+import { useIntl } from 'utils/cl-intl';
+
+import messages from '../../../messages';
+
+type Props = {
+  rawValue: any;
+  customField: IIdeaCustomField;
```

Comment:
The existing type for rawValue is any, and it's used across all the customer fields :thinking: I don't have time now to try and add stricter typing here unfortunately. 
![image](https://github.com/user-attachments/assets/7975a877-119b-4225-86a5-650eb894a85e)